src/frontend/src/components/common/List/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ChangeEvent, MouseEventHandler } from 'react';
+import { faPlus, faTrash } from '@fortawesome/free-solid-svg-icons';
+import InputText from '../InputText/InputText';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+interface InputTextProps {
+  label: string;
+  field: string;
+  value: string;
+  id: string;
+}
+
+interface ListProps {
+  data: InputTextProps[][];
+  columnsName: string[];
+  handleOnChange: (
+    event: ChangeEvent<HTMLInputElement>,
+    row: number,
+    col: number,
+  ) => void;
+  addRow: MouseEventHandler<HTMLButtonElement>;
+  removeRow: (idx: number) => void;
+}
+
+const List = ({
+  data,
+  columnsName,
+  handleOnChange,
+  addRow,
+  removeRow,
+}: ListProps) => {
+  return (
+    <div>
+      <div className="container">
+        <div>
+          <div className="data-table__container ">
+            <table id="tab_logic" className="table data-table">
+              <thead>
+                <tr>
+                  {columnsName.map((column, index) => (
+                    <th className="" key={index}>
+                      {column}
+                    </th>
+                  ))}
+                  <th />
+                </tr>
+              </thead>
+
+              <tbody>
+                {data.map((items, idx) => (
+                  <tr key={idx}>
+                    {items.map((item, index) => (
+                      <td key={index}>
+                        <div className=" row">
+                          <div>
+                            {index === 0 ? (
+                              <label className="mt-3 ">
+                                {'Risk' + ' ' + (idx + 1)}
```

Comment:
Use of this hard-coded string 'Risk' makes this much less reusable.

Reply:
yes, i will change this. will pass label as props to list